src/mscorlib/corefx/System/Globalization/Calendar.cs

```diff
@@ -29,7 +30,7 @@ namespace System.Globalization
     // The calculation of hour/minute/second is moved to Calendar from GregorianCalendar,
     // since most of the calendars (or all?) have the same way of calcuating hour/minute/second.
 
-
+    [Serializable]
```

Comment:
I searched for all serializable types in globalization and then went through and annotated the corresponding ones here.